packages/web-twig/src/Resources/components/Link/Link.stories.twig

```diff
@@ -10,4 +10,12 @@
         {% include '@components/Link/stories/LinkColors.twig' %}
     </DocsSection>
 
+    <DocsSection title="Disabled">
+        {% include '@components/Link/stories/LinkDisabled.twig' %}
+    </DocsSection>
+
+    <DocsSection title="Underline">
```

Comment:
I would use these sections to separate demos and wouldn't add titles inside of them.

Reply:
Do you mean the h3 subtitles?
<img width="278" alt="image" src="https://github.com/user-attachments/assets/9571dea8-008c-41f3-9ed9-3fac61abb6ef">